src/rebar3_codecov_prv.erl

```diff
@@ -77,13 +107,25 @@ format_array_to_list(Module, CallsPerLineArray, Acc) ->
     BinPath = list_to_binary(get_source_path(Module)),
     [{BinPath, ListOfCallTimes}|Acc].
 
+module_to_lcov(Module, CallsPerLineArray, LCovFile) ->
+    io:format(LCovFile, "SF:~s~n", [get_source_path(Module)]),
+    CallTimes = array:to_orddict(CallsPerLineArray),
+    InstrumentedLines = [{N, C} || {N, C} <- CallTimes, C =/= null],
+    ExecutedLines = [{N, C} || {N, C} <- InstrumentedLines, C > 0],
+    [io:format(LCovFile, "DA:~p,~p~n", [N, C]) || {N, C} <- InstrumentedLines],
+    io:format(LCovFile, "LH:~p~n", [length(ExecutedLines)]),
+    io:format(LCovFile, "LF:~p~n", [length(InstrumentedLines)]),
+    io:format(LCovFile, "end_of_record~n", []),
```

Comment:
Could you add a reference to the format specification you used here? It is not obvious for me. Commit message should be enough.

Reply:
added as code comments